metadata/src/from_into/v15.rs

```diff
@@ -330,9 +332,23 @@ mod into_v15 {
 
     fn from_extrinsic_metadata(e: ExtrinsicMetadata) -> v15::ExtrinsicMetadata<PortableForm> {
         v15::ExtrinsicMetadata {
-            version: e.version,
+            // V16 and above metadata can have multiple supported extrinsic versions. We have to
+            // pick just one of these if converting back to V14/V15 metadata.
+            //
+            // - Picking the largest may mean that older tooling won't be compatible (it may only
+            //   check/support older version).
+            // - Picking the smallest may mean that newer tooling won't work, or newer methods won't
+            //   work.
+            //
+            // Either could make sense, but we keep the oldest to prioritize backward compat with
+            // older tooling.
+            version: *e
```

Comment:
does this mean in practice that we will probably just use transaction v4 if we assume both formats are supported by the node we connected to?

Reply:
Yup; we pick the lowest version for now because we know it'll work!

(for instance, right now you can submit v5 extrinsics but the `VerifySignature` extension doesn't appear to be used yet anywhere, which means you couldn't sign them, so they would fail anyways)